Supporting Resources/v2.0/Datamodel/Datafiles/Activity I-SB-1-A.yml

```diff
@@ -28,11 +28,14 @@ shortDescription:
 
 #A multi-paragraph description of the activity
 longDescription: 
-  Define the build process, breaking it down into a set of clear instructions to either be followed by a person or an automated tool. The build process definition desribes the whole process end-to-end so that the person or tool can follow it consistently each time and produce the same result. The definition is stored centrally and accessible to any tools or people. Do not store or distribute multiple copies, some of which may become outdated.
+  Define the build process, breaking it down into a set of clear instructions to either be followed by a person or an automated tool. The build process definition describes the whole process end-to-end so that the person or tool can follow it consistently each time and produce the same result. The definition is stored centrally and accessible to any tools or people. Avoid storing multiple copies of the definition as they may become unaligned and outdated.
```

Comment:
"of the definition" in the last sentence seems superfluous. Maybe Pat can help

Reply:
Done